app/navs/documentation.js

```diff
@@ -168,7 +168,7 @@ export const documentationNav = [
   {
     title: (
       <Title
-        title="Deploying to Production"
+        title="Uitrollen naar Productie"
```

Comment:
`Productie` met kleine letter (hoewel het qua zicht in dit geval niet uitmaakt :)